net/queries/useWeb2Signup.ts

```diff
@@ -0,0 +1,15 @@
+import client from "../client";
+import { MutationBuilder } from "./builder";
+
+export interface Web2SignupRequest {
+  email: string;
+  username: string;
+  password: string;
+}
+
+const useWeb2Login = new MutationBuilder<Web2SignupRequest>(client)
```

Comment:
Rename this to useWeb2SignUp as suggested by the bot.

Reply:
oops, will fixup.